CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- Add `skip_emscripten_version_check` flag to skip emscripten version check.
```

Comment:
```suggestion
- Add `skip_emscripten_version_check` flag and SKIP_EMSCRIPTEN_VERSION_CHECK environment
variable to skip emscripten version check.
```

Since we don't have docs for `pyodide-build` deployed yet, maybe this would be more explicit/helpful?

Reply:
Sounds good.